packages/web-api/src/WebClient.ts

```diff
@@ -667,13 +688,11 @@ export class WebClient extends Methods {
    * a string, used when posting with a content-type of url-encoded. Or, it can be a readable stream, used
    * when the options contain a binary (a stream or a buffer) and the upload should be done with content-type
    * multipart/form-data.
-   * @param options - arguments for the Web API method
-   * @param headers - a mutable object representing the HTTP headers for the outgoing request
+   * @param config - The Axios request configuration object
    */
-  private serializeApiCallOptions(
-    options: Record<string, unknown>,
-    headers?: Record<string, string>,
-  ): string | Readable {
+  private serializeApiCallOptions(config: InternalAxiosRequestConfig): InternalAxiosRequestConfig {
```

Comment:
A nit here: I see you had to alias the original `options` variable to `data`. I sort of agree that "API call options" makes less sense in this context. To me, request `data` makes more sense than request `options` (and maybe even could be confused with HTTP OPTIONS method). I would be in favour of renaming this private method to `serializeApiCallData` to make this clearer.

Reply:
`options` alias has been removed and private method has been renamed to `serializeApiCallData`